NeuralAmpModeler/NeuralAmpModeler.h

```diff
@@ -37,6 +37,7 @@ enum EParams
   kToneTreble,
   kOutputLevel,
   // The rest is fine though.
+  kNamToggle,
```

Comment:
[Critical] Adding parameters breaks backward compatibility when the plug-in unserializes. I need to make sure that when users update their plug-in that their old sessions still load up correctly.

There's an example of how to handle it here: #487.

Since this will add a new parameter to the plug-in, there needs to be a "legacy" unserialization implemented that's similar to this: https://github.com/sdatkinson/NeuralAmpModelerPlugin/blob/45484dc589a474f05ffc80e328e71cb7145a7ff7/NeuralAmpModeler/NeuralAmpModeler.cpp#L823 but for version 0.7.10.

And logic to make sure that old versions of the plug-in load up correctly needs to be added here: https://github.com/sdatkinson/NeuralAmpModelerPlugin/blob/45484dc589a474f05ffc80e328e71cb7145a7ff7/NeuralAmpModeler/NeuralAmpModeler.cpp#L411

Reply:
Right. I can see that 0_7_9 and 0_7_10 have same parameters.
If I am right then __UnserializeStateLegacy_0_7_9_ function can be reused - I did it in my latest commit.
But since I rely on unserialized version to decide whether __UnserializeStateLegacy_0_7_9_ or __UnserializeStateCurrent_ it's required to upgrade current NAM version to 0.7.11.
I don't include upgrade in my commit.
I also ran tests provided by Steinberg, there's one still failing due to above check, once version upgrade is done it should pass.

Reply:
> If I am right then _UnserializeStateLegacy_0_7_9 function can be reused - I did it in my latest commit.

Agreed. Let me bump the patch version and merge that in. Then you can merge that into your work and we should be able to expect it to work.